pkgs/misc/uboot/default.nix

```diff
@@ -394,4 +394,15 @@ in {
     extraMeta.platforms = ["armv7l-linux"];
     filesToInstall = ["u-boot.img" "SPL"];
   };
+
+  ubootRockPi4 = buildUBoot {
+    defconfig = "rock-pi-4-rk3399_defconfig";
+    extraMeta.platforms = ["aarch64-linux"];
+    BL31 = "${armTrustedFirmwareRK3399}/bl31.elf";
+    filesToInstall = [ "u-boot.itb" "idbloader.img"];
+    postBuild = ''
+      ./tools/mkimage -n rk3399 -T rksd -d tpl/u-boot-tpl.bin idbloader.img
```

Comment:
Something smells fishy here. The `idbloader.img` isn't produced by that defconfig like every other rk3399 defconfig?

Reply:
I guess this comment can be resolved as the code I had here was just duplicating efforts?